components/coins-status/index.js

```diff
@@ -1,43 +1,110 @@
-import { useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import { useAnimateValue } from './useCoinStatus';
 import styles from './coin.module.css';
 import PropTypes from 'prop-types';
 
+const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
+const WALLET_URL = `${BASE_API_URL}/wallet`;
+const CURRENCIES = {
+  NEELAM: 'neelam',
+  DINERO: 'dinero',
+};
```

Comment:
Can we have these constants in `constants.js` file in the root? Will require them in other components as well.

Reply:
moved to constants.js file